### PR TITLE
Echo y to the gem update command to accept bundle executable overwrite prompt.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

**Comment by zach-klippenstein** [comment]
> Seems fine 🤷‍♂

image

Computer says no.

**Comment by JustinDSN** [comment]
I would try removing the `gem update --system` as we could probably use the Travis provided bundler that's installed.

**Comment by zach-klippenstein** [comment]
> I would try removing the gem update --system as we could probably use the Travis provided bundler that's installed.

See the commits in this PR – I tried that and I tried adding `--force --quiet` flags, and neither of those worked. The last commit just echos `y` to the command, if that doesn't work I give up for now.

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ matrix:
       os: osx
       osx_image: xcode10.2
       install:
-        - gem update --system
+        - echo y | gem update --system
```
Comment (AquaGeek) [inline .travis.yml]: `yes | gem update --system`?

Comment (zach-klippenstein) [reply .travis.yml]: 😲

Comment (zach-klippenstein) [reply .travis.yml]: Done

**Comment by zach-klippenstein** [comment]
Re-requested Tyler's review before seeing he'd already looked at this again, so I'm going to merge once it goes green.

**Pull request merged**